src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -1065,7 +1036,7 @@ namespace HeatBalFiniteDiffManager {
 
     int numNodesInMaterialLayer(EnergyPlusData &state, std::string const &surfName, std::string const &matName)
     {
-        for (auto &surface : state.dataSurface->Surface) {
+        for (auto const &surface : state.dataSurface->Surface) {
```

Comment:
[src/EnergyPlus/HeatBalFiniteDiffManager.cc:1068]:(style),[constVariable],Variable 'surface' can be declared as reference to const